telegram/constants.py

```diff
@@ -1754,3 +2587,50 @@ class ForumTopicLimit(IntEnum):
     * :paramref:`~telegram.Bot.edit_general_forum_topic.name` parameter of
       :meth:`telegram.Bot.edit_general_forum_topic`
     """
+
+
+class Date(Enum):
+    """This enum contains the date literal for :class:`telegram.InaccessibleMessage`
+
+    .. versionadded:: NEXT.VERSION
+    """
+
+    __slots__ = ()
+
+    ZERO_DATE = datetime.datetime(1970, 1, 1, tzinfo=UTC)
```

Comment:
just double checking: is this enough to make the object properly tz-aware in the pytz-sense? Otherwise we could use something like 

https://github.com/python-telegram-bot/python-telegram-bot/blob/a8f1164b0cb9bb1e26920a03ae5c88b167bd3041/tests/_utils/test_datetime.py#L58-L62

Reply:
According to https://stackoverflow.com/a/50710825 at least.